addons/mail/static/src/model/model_manager.js

```diff
@@ -166,7 +166,16 @@ export class ModelManager {
      */
     all(Model, filterFunc) {
         for (const listener of this._listeners) {
-            this._listenersObservingAllByModel.get(Model).add(listener);
+            const entry = this._listenersObservingAllByModel.get(Model);
+            const info = {
+                reason: `all() - ${Model}`,
+                stack: Error().stack,
+            };
+            if (entry.has(listener)) {
+                entry.get(listener).push(info);
+            } else {
+                entry.set(listener, [info]);
+            }
```

Comment:
remove from this pr

Reply:
Do you mean whole commit that make changes to model manager?

Reply:
Yes as we made a separate PR for it (either that or it will be included in mine, but it shouldn't be here either way).